tests/test_wellbore_marker_frame.py

```diff
@@ -0,0 +1,433 @@
+                                                   feature_name = 'horizon_feature_1')
+    horizon_feature_1.create_xml()
+    horizon_interp_1 = rqo.HorizonInterpretation(parent_model = model,
+                                                 title = 'horizon_interp_1',
+                                                 genetic_boundary_feature = horizon_feature_1,
+                                                 sequence_stratigraphy_surface = 'flooding',
+                                                 boundary_relation_list = ['conformable'])
+    horizon_interp_1.create_xml()
+
+    horizon_feature_2 = rqo.GeneticBoundaryFeature(parent_model = model,
+                                                   kind = 'horizon',
+                                                   feature_name = 'horizon_feature_2')
+    horizon_feature_2.create_xml()
+    horizon_interp_2 = rqo.HorizonInterpretation(parent_model = model,
+                                                 title = 'horizon_interp_2',
+                                                 genetic_boundary_feature = horizon_feature_1,
+                                                 sequence_stratigraphy_surface = 'transgressive',
+                                                 boundary_relation_list = ['conformable'])
+    horizon_interp_2.create_xml()
+    horizon_interp_2_uuid = horizon_interp_2.uuid
+
+    fault_feature_1 = rqo.TectonicBoundaryFeature(parent_model = model,
+                                                  kind = 'fault',
+                                                  feature_name = 'fault_feature_1')
+    fault_feature_1.create_xml()
+    fault_interp_1 = rqo.FaultInterpretation(parent_model = model,
+                                             title = 'fault_interp_1',
+                                             tectonic_boundary_feature = fault_feature_1,
+                                             is_normal = True,
+                                             maximum_throw = 15)
+    fault_interp_1.create_xml()
+
+    wellbore_marker_frame_dataframe = pd.DataFrame({
+        'MD': [300, 310, 330],
+        'Type': ['horizon', 'horizon', 'fault'],
+        'Surface': ['marker_horizon_1', 'marker_horizon_2', 'marker_fault_1'],
+        'Interp_Surface': ['horizon_interp_1', 'horizon_interp_2', 'fault_interp_1'],
+        'Well': well_name
+    })
+
+    # Create a wellbore marker frame from a dataframe
+    wellbore_marker_frame = resqpy.well.WellboreMarkerFrame.from_dataframe(parent_model = model,
+                                                                           dataframe = wellbore_marker_frame_dataframe,
+                                                                           trajectory = trajectory)
+    # --------- Act ----------
+    # Get an interpretation based on an uuid
+    found_interpretation = wellbore_marker_frame.get_interpretation_obj(interpretation_uuid = horizon_interp_2_uuid)
```

Comment:
This is fine for now as a test. Could you please add to the comment to say that this method will have breaking changes (or be removed) sometime soon!